test/mixins.py

```diff
@@ -209,3 +211,12 @@ def bigflow_run(self, cmd, **kwargs):
     def bigflow_spawn(self, cmd, **kwargs):
         cmd = ["python", "-m", "bigflow", *cmd]
         return self.subprocess_spawn(cmd, **kwargs)
+
+
+class ABCTestCase(unittest.TestCase):
+
+    __test__ = True
```

Comment:
Why do we need this property?

Reply:
It allows to disable unittest classes by adding '__test__ = False'
But tests are enabled by default.